src/styles/index.scss

```diff
@@ -2,2 +1,115 @@
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
```

Comment:
The use of `&__img` suggests that `.card__img` is an element of `.card`, but it's not clear if the `.card` block is defined in this file or another. Ensure that the `.card` block exists to maintain BEM structure.